resources/migrations/001_update_migrations.yaml

```diff
@@ -4381,6 +4381,21 @@ databaseChangeLog:
                     nullable: true
                   remarks: 'If true, the table requires a filter to be able to query it'
 
+  - changeSet:
+      id: v49.00-012
+      author: qnkhuat
+      comment: Add metabase_field.database_partitioned
+      changes:
+        - addColumn:
+            tableName: metabase_field
+            columns:
+              - column:
+                  name: database_partitioned
+                  type: boolean
+                  constraints:
+                    nullable: true
```

Comment:
Why did you decide to make this field nullable, rather than adding a default false value like `database_is_auto_increment`?

I prefer non-nullable constraints unless there's a reason not to, because it eliminate the possibility of accidentally handling `nil` values differently from `false` values.

Reply:
not all databases support partition, so for fields from those databases, it should be `null.` it's in the same vein as `database_indexed`.

Reply:
I don't understand your argument, but I guess there's a possible potential future where we want to handle these two cases differently:
- fields of databases that support partitions
- fields of databases that don't support partitions

So it might be helpful to differentiate them at the field level. I'm not sure either way so I'm okay with this